libs/librepcb/core/project/board/drc/boarddesignrulecheckdata.h

```diff
@@ -63,13 +64,16 @@ struct BoardDesignRuleCheckData final {
     Point position;
     PositiveLength size;
     PositiveLength drillDiameter;
+    // Only filled with the layers of traces directly connected to the via.
+    QSet<const Layer*> connectedLayers;
     const Layer* startLayer;
     const Layer* endLayer;
     std::optional<std::pair<const Layer*, const Layer*>> drillLayerSpan;
     bool isBuried;
     bool isBlind;
     std::optional<PositiveLength> stopMaskDiameterTop;
     std::optional<PositiveLength> stopMaskDiameterBot;
+    NetSignal* netSignal;  // May be null.
```

Comment:
Unfortunately this pointer should not be added to the struct as it is not safe to be accessed from threads. I know currently it is never dereferenced and thus won't cause problems, but honestly it feels dangerous to have it here (easy to access it in future while forgetting its non-threadsafety). Fortunately we don't need it anyway - just use `Segment::net` instead which is already there and fully threadsafe.